Make documentation clear for non-privileged users (#12)
stfc/openstack-query-library: docs/user_docs/USAGE.md

> # Run - this will run the query in the "prod" cloud domain and with meta-params as_admin and all_projects.
> # This will run the query as an admin and will find VMs on every available project

Reviewer: ```suggestion
# Run - this will run the query in the "prod" cloud domain and will find VMs in your project set in `clouds.yaml`.
```
